server/types/express.d.ts

```diff
@@ -1,13 +1,13 @@
 import DataLoader from 'dataloader';
 
 import PersonalToken from '../models/PersonalToken';
-import User from '../models/User';
+import UserModel from '../models/User';
 import UserToken from '../models/UserToken';
 
 declare global {
   namespace Express {
     interface Request {
-      remoteUser?: User | null;
```

Comment:
Had to change this because we now have `@types/passport` in our dependency tree and it messes up with the global `Express.Request.User` namespace: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/40a9d006062d9dfe259599d73b8f5a091de487a6/types/passport/index.d.ts#L20.